submissions/ustymchyk/friends-app/css/cards.css

```diff
@@ -0,0 +1,45 @@
+.cards {
+  grid-area: cards;
+
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+  gap: 20px;
+
+  max-width: 1400px;
+  margin: 0 auto;
+  padding: 0 20px 20px 20px;
+
+  list-style: none;
+}
+
+.card {
```

Comment:
I'd recommend you decreasing the size of the cards because now they look extremely big